crates/air_r_formatter/src/lib.rs

```diff
@@ -209,6 +209,10 @@ where
         f.context().comments().is_suppressed(node.syntax())
     }
 
+    fn fmt_suppressed(&self, node: &N, f: &mut RFormatter) -> FormatResult<()> {
```

Comment:
```suggestion
    /// Formats a suppressed node
    ///
    /// You may want to override this method if you need to manually handle
    /// formatting of a suppressed node. This should be extremely rare.
    fn fmt_suppressed(&self, node: &N, f: &mut RFormatter) -> FormatResult<()> {
```

To match the other `fmt_*()` methods with defaults in the trait here